fsd_config/form_jsons/dpi_r2/name-your-application-dpi.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
This looks basically identical to all the other name your application forms - could we reuse one of them instead of redefining?

Reply:
So unfortunately the others have the fund short name as the suffix, so the link on the form will look like /name-your-application-cyp/.

What ill do is ill keep this particular form without the suffix, so we have a reusable one